File: src/main/java/com/bz/springkafka/spring_kafka/AppStorage.java
package com.bz.springkafka.spring_kafka;


import lombok.AccessLevel;
import lombok.NoArgsConstructor;

import java.util.Map;
import java.util.concurrent.ConcurrentHashMap;

@NoArgsConstructor(access = AccessLevel.NONE)
public class AppStorage {

    private static final Map<String, String> STORAGE = new ConcurrentHashMap<>();

    public static void add(String k, String v) {
        STORAGE.put(k, v);
    }

    public static String get(String k) {
        return STORAGE.get(k);
    }

}

File: src/main/kotlin/com/bz/springkafka/storage/AppStorage.kt
package com.bz.springkafka.storage

import java.util.concurrent.ConcurrentHashMap

object AppStorage {
    private val STORAGE: MutableMap<String, String> = ConcurrentHashMap()

    fun add(k: String, v: String) {
        STORAGE[k] = v
    }

    fun get(k: String): String? {
        return STORAGE[k]
    }

}
